process/elasticproc/block/blockProcessor_test.go

```diff
@@ -434,7 +434,7 @@ func TestBlockProcessor_PrepareBlockForDBMiniBlocksDetails(t *testing.T) {
 	require.Equal(t, &data.Block{
 		Hash:            "68617368",
 		Size:            int64(723),
-		SizeTxs:         94,
+		SizeTxs:         0,
```

Comment:
Why is this not computed anymore here?

Reply:
added transaction in the pool in order to compute the size.